manager/rh_status_handler.py

```diff
@@ -0,0 +1,48 @@
+"""
+Redhat status page API
+"""
+import os
+import signal
+from asyncio import ensure_future
+from aiokafka.errors import KafkaError
+from peewee import PeeweeException
+
+from common.mqueue import MQReader, EVALUATOR_TOPIC
+from .base import GetRequest, RhStatusTimeoutException
+
+TIMEOUT_SECS = int(os.getenv("STATUS_TIMEOUT", "10"))
+
+
+class GetRhStatus(GetRequest):
+    """Class for handling the rh status requests"""
+
+    @staticmethod
+    def _alarm_handler(signal_num, _):
+        """Handler for SIGALRM"""
+        if signal_num == signal.SIGALRM:
+            raise RhStatusTimeoutException("API timed out")
+
+    @staticmethod
+    def _close_kafka_sync(reader):
+        """Close the aiokafka reader in sync context"""
+        ensure_future(reader.stop())
+
+    @classmethod
+    def handle_get(cls, **kwargs):  # pylint: disable=unused-argument
+        """Meassure the status of the API, db"""
+        signal.signal(signal.SIGALRM, cls._alarm_handler)
+        signal.alarm(TIMEOUT_SECS)
+
+        # Check if db is alive, API is responding in time,
+        # Kafka vulnerability topic is alive
+        reader = None
+        try:
+            # Db is already connected from middlewares
+            reader = MQReader([EVALUATOR_TOPIC])
```

Comment:
I think it's not good idea, creating consumer and then immediately closing it. Attaching new consumer trigger rebalancing on server and attaching partition to this consumer and he immediately throws it away triggering another rebalance.

Reply:
Yeah, you are right, I will try to change it